integration-tests/go/aws-sdk/main.tf

```diff
@@ -0,0 +1,14 @@
+module "test" {
+  source = "../../../opentelemetry-lambda/go/integration-tests/aws-sdk"
+
+  enable_collector_layer = true
+  collector_layer_name = var.collector_layer_name
```

Comment:
I think this confirms my above fear that both [.NET Collector Layer Name](https://github.com/open-telemetry/opentelemetry-lambda/blob/db6ff3a488e0d54f2f979f7540c5a5baccef5fe3/dotnet/integration-tests/aws-sdk/wrapper/variables.tf#L4) and [Go Collector Layer Name](https://github.com/open-telemetry/opentelemetry-lambda/blob/db6ff3a488e0d54f2f979f7540c5a5baccef5fe3/go/integration-tests/aws-sdk/variables.tf#L4) will use the same collector layer and will create it with Terraform during Soak Tests. What do you think?